binding.gyp

```diff
@@ -158,7 +158,7 @@
 			]
 		},
 		{
-			"target_name": "<(module_name)",
+			"target_name": "bindings",
```

Comment:
Is there a specific thing about removing module_name here? When keeping it `require('bindings')('ext2fs')` is a bit clearer – or it could be statically set to something more descriptive?